code/game/gamemodes/changeling/powers/lesserform.dm

```diff
@@ -21,6 +21,8 @@
 		to_chat(H, "<span class='warning'>We cannot perform this ability in this form!</span>")
 		return
 
+	var/datum/dna/DNA = changeling.GetDNA(user.dna.real_name)
+	user.dna = DNA ? DNA.Clone() : user.dna
```

Comment:
> This particular part was important as otherwise when we set the monkeyblock we'd be setting the monkeyblock of the actual genome we've got in our repository, so when we switched back and re-inherited all the SE features of said genome when we humanform, we'd be stuck a monkey

It *does* set that gene, but going back into human form explicitly unsets it anyway, so it didn't actually cause any problems... but it's better off not mucking with the stored genome.

If you just want to make sure the DNA gets cloned, though, can't you just do `user.dna = user.dna.Clone()`?